ppsci/geometry/mesh.py

```diff
@@ -127,6 +126,8 @@ def inflated_random_points(self, n, distance, random="pseudo"):
             raise ValueError(
                 f"len(n)({len(n)}) should be equal to len(distance)({len(distance)})"
             )
+        from ppsci.geometry import inflation
+
```

Comment:
why support gemetry here.

Reply:
Maybe wrap this into a funcition, and raise a friendly exception message, such as "Inflation backend hasn't installed, please install it or use other random points."

Reply:
Done with `importlib`